.bcr/config.yml

```diff
@@ -0,0 +1,3 @@
+fixedReleaser:
```

Comment:
The publish-to-bcr app needs to know who the releaser is in order to:
* send error emails
* tag their username in the bcr PR
* make them the commit author

When a repository uses release automation and the author of a release is the `github-actions` bot (rules_python does this), then the user who caused the release is obscured. `fixedReleaser` also helps in the case where a user is the releaser author, but their email is hidden from their profile, so the app can't access it to send emails or properly attribute the commit to them.

Reply:
So if you "push a tag to release", we can never fix this? What if you had to push an annotated tag to release, so that there's a commit-like object in a git ref?

Reply:
The problem lies with the GitHub api. It uses the identity of whatever authorized token is used to create the release to set as the author. It will not look at annotated tag information and there are no fields to explicitly set the author. So if the actions GITHUB_TOKEN is used to create the release, there's nothing we can really do unless they expand the api.

Reply:
okay well this isn't the PR to dig into that anyhow :)